### PR TITLE
v0.9.0: prepare for v1 release

### PR DESCRIPTION
**Pull request by brenns10**
This pull request prepares the v1 release. I decided to create v0.9.0 so I could build some RPMs for internal testing, and we can merge any fixes prior to tagging v1.0.0.

Main items of this pull request:

- Move the v2 modules into `drgn_tools/v2`
- Add RPM packaging which removes this directory prior to installation
- Add manual page

I did include a couple commits which improve corelens CLI. I did these as I wrote the manual page:

- Put the timing information into the `report/corelens` file now, so we can see it in a report.
- The `corelens -L` and `corelens -M module -h` now show information about which kernel modules are required, and whether the module runs for `-A` only, or manually only.
- Add a blanket warning for running Corelens on live systems
- Check the CTF compatibility prior to loading it, to avoid loading known-buggy CTF.

And then a couple things I included because they were in my development branches and they're fine to include here:

- Update to the `drgn_tools.cli` (i.e. the DRGN command, which is not included in the RPM) to allow it to find CTF debuginfo automatically. I've found this useful a few times already.
- Fix crash in the debuginfo downloader if there is a DNS resolution failure. This happens for me when I disconnect from the VPN... but frequently `oss.oracle.com` is faster off-VPN than the internal site :)

### CODE REVIEW

**Review by biger410 — COMMENTED** [review]
The changes look good. The only question is that 0.9 is binding with drgn 0.0.25, will 1.0 be binding with 0.0.26?

File: buildrpm/python-drgn-tools.spec [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+BuildRequires:  python%{python3_pkgversion}-pip
+BuildRequires:  python%{python3_pkgversion}-wheel
+
+%global python_wheelname drgn_tools-%{version}-py3-none-any.whl
+
+# The drgn dependency can be fulfilled by drgn with, or without, CTF support.
+# However, drgn-tools is tied to a specific drgn release.
+Requires:       drgn >= 0.0.25, drgn < 0.0.26
```
[the remark this answers is not in the document]
Comment (brenns10) [reply buildrpm/python-drgn-tools.spec:23]: What may make sense for this interim release is to have this support 0.0.25 and 0.0.26, but then have drgn-tools 1.0 require drgn 0.0.26.

[the remark this answers is not in the document]
Comment (biger410) [reply buildrpm/python-drgn-tools.spec:23]: LGTM

**Comment by brenns10** [comment]
@biger410 I went ahead and updated the drgn required versions, and added a tweak to the manual page. Should be ready for merge now.

**Review by biger410 — APPROVED** [review]

**Pull request merged**